### PR TITLE
K8SPSMDB-233 New users check

### PR DESCRIPTION
**Pull request by nonemax**
[K8SPSMDB-233](https://jira.percona.com/browse/K8SPSMDB-233)

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| one-pod | passed |
| init-deploy | passed |
| storage | passed |
| upgrade | passed |
| monitoring | passed |
| self-healing | passed |
| limits | passed |
| upgrade-consistency | passed |
| scaling | passed |
| monitoring-2-0 | failed |
| demand-backup | failed |
| arbiter | passed |
| security-context | passed |
| operator-self-healing | passed |
| scheduled-backup | passed |
| service-per-pod | passed |
| liveness | passed |
| users | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| one-pod | passed |
| init-deploy | passed |
| storage | passed |
| upgrade | passed |
| monitoring | passed |
| self-healing | failed |
| limits | passed |
| operator-self-healing | passed |
| upgrade-consistency | passed |
| scaling | passed |
| monitoring-2-0 | passed |
| demand-backup | passed |
| security-context | passed |
| arbiter | passed |
| scheduled-backup | passed |
| service-per-pod | passed |
| liveness | passed |
| users | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| one-pod | passed |
| storage | passed |
| upgrade | passed |
| monitoring | passed |
| self-healing | failed |
| limits | passed |
| operator-self-healing | passed |
| upgrade-consistency | passed |
| scaling | passed |
| monitoring-2-0 | failed |
| demand-backup | passed |
| arbiter | failed |
| security-context | passed |
| scheduled-backup | passed |
| service-per-pod | passed |
| liveness | passed |
| users | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| one-pod | passed |
| init-deploy | passed |
| upgrade | passed |
| storage | passed |
| monitoring | passed |
| limits | passed |
| self-healing | passed |
| upgrade-consistency | passed |
| scaling | failed |
| monitoring-2-0 | failed |
| demand-backup | passed |
| security-context | passed |
| arbiter | passed |
| operator-self-healing | passed |
| scheduled-backup | passed |
| service-per-pod | passed |
| liveness | passed |
| users | passed |

File: pkg/controller/perconaservermongodb/psmdb_controller.go [hunk 1]
```diff
@@ -130,8 +131,9 @@ type ReconcilePerconaServerMongoDB struct {
 	serverVersion *version.ServerVersion
 	reconcileIn   time.Duration
 
-	statusMutex *sync.Mutex
-	updateSync  int32
+	statusMutex            *sync.Mutex
+	updateSync             int32
+	sfsTemplateAnnotations map[string]string
```
Comment (dAdAbird) [inline pkg/controller/perconaservermongodb/psmdb_controller.go:136]: `sfsTemplateAnnotations` doesn't belong here.

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| upgrade | passed |
| upgrade-consistency | passed |
| demand-backup | passed |
| scheduled-backup | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| init-deploy | failed |
| limits | passed |
| self-healing | passed |
| scaling | passed |
| security-context | passed |
| operator-self-healing | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| init-deploy | passed |
| upgrade | passed |
| one-pod | passed |
| self-healing | failed |
| monitoring | passed |
| limits | passed |
| operator-self-healing | passed |
| upgrade-consistency | passed |
| scaling | passed |
| demand-backup | passed |
| monitoring-2-0 | failed |
| security-context | passed |
| arbiter | failed |
| scheduled-backup | passed |
| service-per-pod | passed |
| liveness | passed |
| users | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| one-pod | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| init-deploy | passed |
| limits | passed |
| scaling | passed |
| security-context | passed |
| upgrade | passed |
| upgrade-consistency | passed |
| demand-backup | passed |
| scheduled-backup | passed |
| storage | passed |
| self-healing | passed |
| arbiter | failed |
| service-per-pod | passed |
| liveness | passed |
| users | passed |
| operator-self-healing | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| self-healing | passed |
| operator-self-healing | passed |
| one-pod | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| arbiter | failed |
| init-deploy | passed |
| limits | passed |
| scaling | passed |
| security-context | passed |
| upgrade | passed |
| upgrade-consistency | passed |
| demand-backup | passed |
| scheduled-backup | passed |
| service-per-pod | passed |
| liveness | passed |
| users | passed |

**Comment by JNKPercona** [comment]
PSMDB operator docker - perconalab/percona-server-mongodb-operator:PR-404-f83b4649

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| limits | passed |
| scaling | passed |
| security-context | passed |
| storage | passed |
| self-healing | passed |
| operator-self-healing | passed |
| one-pod | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| arbiter | passed |
| upgrade | passed |
| upgrade-consistency | passed |
| demand-backup | passed |
| scheduled-backup | passed |
| service-per-pod | passed |
| liveness | passed |
| users | passed |

**Pull request merged**